depression-and-anxiety/getting-therapy.md

```diff
@@ -72,3 +72,24 @@ They can help you with things like talking to other people about your problem or
 - [Mind](http://www.mind.org.uk/information-support/types-of-mental-health-problems/depression/)
 - [Rethink mental illness](http://www.rethink.org/diagnosis-treatment/conditions/depression)
 - [Big White Wall](https://www.bigwhitewall.com/landing-pages/landingv3.aspx)
+
+##You can get therapy online
+
+A type of therapy called cognitive behavioural therapy (CBT) is available online and it's may be free on the NHS, depending where you live. 
+
+The benefits of computerised CBT include:
+
+-  no waiting time
+-  you don't a lot of experience on computers to use them
```

Comment:
Ahem, excellent typo correction skills there. That should be:

Should this be "you don't **need** a lot"?